iis-common/src/main/java/eu/dnetlib/iis/common/StaticResourceProvider.java

```diff
@@ -0,0 +1,79 @@
+package eu.dnetlib.iis.common;
+
+import org.apache.commons.io.IOUtils;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StaticResourceProvider {
```

Comment:
I guess we could add a brief javadoc description for this class.

And one minor remark: since all the methods from this class are relying on classpath resources then maybe it would more informative when renaming `StaticResourceProvider` to `ClassPathResourceProvider`. What do you think?

Reply:
+1 for Javadoc
The `StaticResourceProvider` name at least would need some explanation. And I have no intuition if `ClassPathResourceProvider` is right or too much of an implementation-detail name but all the docs below mention classpath, so maybe it would be a better name?

Reply:
Yes, I thought of renaming the class to :) Plus adding class javadoc.